MIPS/mip-0005-mip-upgrade.md

```diff
@@ -0,0 +1,315 @@
+
+### Changes in the Drafting and Review Phases
+
+**Drafting Phase - MIP Submission**
+
+During the Drafting phase, when the MIP Author submits a new MIP as a pull request in GitHub, a MIP Facilitator is assigned to assist the MIP Author, review the MIP and (if necessary, consult with the MIP Reviewers to) decide whether to approve or reject the pull request according to the following acceptance criteria:
+- The proposal is original or new.
+- The proposal is clearly articulated.
+- The proposal uses the MIP Template correctly.
+- The proposal is not missing any required parts.
+
+If the pull request is rejected, the MIP Facilitator will communicate the decision to the MIP Author explaining the reasons for the rejection and a list of changes that could be made for the proposal to be accepted, so that the MIP Author can make these changes.
+
+If the pull request is accepted, the MIP Facilitator:
+- Approves the MIP and assigns it a formal MIP number.
+- Changes the status of the MIP.
+- Merges in the PR.
+- Communicates the decision to the MIP author.
+
+**Review Phase - Part 1: Risk Assessment**
+We propose to create a pool of experts to review MIPs across the 3 categories. For each MIP, the Author reaches out to the pool of experts for the relevant category and sends a Request for Comments. To maintain independence, the Author does not choose who the Reviewers are. This MIP Review Group has a period to review the MIP and provide their feedback. The MIP Author can incorporate the feedback into the MIP until a final draft is completed. If the MIP Author does not incorporate the feedback, the MIP process still continues.
+
+When the MIP Author considers the draft ready for the final review, editing stops for at least 1 week to allow MIP Reviewers to write individually a Risk Assessment that is incorporated into the designated section of the MIP template. 
+
+The MIP Facilitator must check that every MIP has enough reviews to complete the Review phase (at least 3 if there is agreement, 5 otherwise). The MIP Facilitator can also coordinate communications and collaborate with the MIP Author to schedule any necessary meetings, and to define a roadmap with target dates for each milestone in the MIP process. 
+
+**Review Phase - Part 2: Community Conversation with Mass Deliberation**
+The MIP Facilitator sets a review period for the community to review the MIP, including the Risk Assessment, and schedules an open call where the MIP Author and MIP Review Group can present the proposal in detail. During this call, they will discuss the risks, benefits, and tradeoffs, address questions from the community, and provide instructions for on-chain voting, including the voting dates.
```

Comment:
This process does not anticipate or provide for incorporation of community feedback. There should be a feedback loop / opportunity to address community concerns not already raised by the expert reviewers, if any.

Reply:
You are right that this upgrade does not explicitly address a formal feedback loop for community comments after the expert review phase. However, this could certainly be explored and proposed in a future MIP.

For clarity, it's important to note that the current process allows for community members to leave feedback on platforms like Mina Research or GitHub at any point before the on-chain vote, and this will not change with this proposal.

What this upgrade introduces is a community call where the MIP Author and Reviewers explain the final proposal, including the risk assessment, to provide the community with a better understanding of what they are voting on. This ensures that everyone has the chance to ask questions and clarify concerns.